Documentation/deploy-master-single.md

```diff
@@ -112,7 +112,7 @@ spec:
     - --service-cluster-ip-range=${SERVICE_IP_RANGE}
     - --secure_port=443
     - --advertise-address=${ADVERTISE_IP}
-    - --admission-control=NamespaceLifecycle,NamespaceExists,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
+    - --admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
```

Comment:
Also, only sort of related, but all "master" components use the insecure port. Which bypasses all auth checks (and I'm pretty sure admission controller checks, but I'd have to trace code to make sure).